photon-icons/photon-icons.json

```diff
@@ -0,0 +1,36 @@
+{
+  "license": "This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.",
+  "fills": [
+    "blue-40",
+    "blue-50",
+    "context-fill",
```

Comment:
If we define all the colors that can be used as fill what are the cases when `content-fill` is still needed?

Reply:
`context-fill` would still be needed for icons which react to the theme they're used in. (Like for add-ons or toolbar buttons.)